### PR TITLE
currentPricePerTonne added to updatePairPrice + unit test

### PR DESCRIPTION
**Pull request by psparacino**
The initial values were being set however updatePairPrice also needed to be modified to set the updated currentPricePerTonne.

Added unit test as well.

### CODE REVIEW

File: pairs/src/pair.utils.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    let decimalRatio = BigDecimal.fromString(cco2_contract.decimalRatio().toString())
+    let burningPercentage = BigDecimal.fromString(cco2_contract.burningPercentage().toString())
+  
+    /** Need to take into the account the fee, charged in the retired token
+     * i.e. if the goal is to retire 100 tonnes at a 10% fee
+     * ex. 100 = .9 * x. x = 111.11
+     * So cost to the user is the same as retiring 111.11 tonnes, which accounts for the fee and actually retires 100 tonnes
+     */
+  
+    let effectiveFeePercentage = BigDecimal.fromString('1').minus(burningPercentage.div(decimalRatio))
```
Comment (biwano) [inline pairs/src/pair.utils.ts:27]: This fee exists only for CCO2?

**Review by biwano — APPROVED** [review]
A question about the CCO2 fees

File: pairs/src/pair.utils.ts (hunk 1 above)
Comment (psparacino) [reply pairs/src/pair.utils.ts:27]: yes, this fee is specific to cco2 and uses the values from the contract that cco2 uses to calculate the fee

**Review by 0xAurelius — APPROVED** [review]
Love the matchstick tests, keep it up ❤️

**Pull request merged**